features/dashboards/components/WidgetView.tsx

```diff
@@ -0,0 +1,121 @@
+import { InformationCircleIcon } from "@heroicons/react/outline";
+import { Tooltip } from "@chakra-ui/react";
+import { Widget } from "@prisma/client";
+import {
+  activeWidgetNameSelector,
+  setActiveWidgetName,
+} from "@/features/records/state-slice";
+import { isUndefined } from "lodash";
+import { useAppDispatch, useAppSelector } from "@/hooks";
+import { useRouter } from "next/router";
+import React, { useMemo } from "react";
+import Shimmer from "@/components/Shimmer";
+import classNames from "classnames";
+
+export type WidgetOptions = {
+  prefix: string;
+  suffix: string;
+};
+
+function WidgetView({
```

Comment:
Why isn't this called `Widget`? Usually `SOMETHING View` is the page that holds the something.